action.yml

```diff
@@ -20,7 +20,7 @@ description: |-
 inputs:
   service:
     description: |-
-      (Required, unless providing `metadata`) ID of the service or
+      (Required, unless providing `metadata` or `job`) ID of the service or
```

Comment:
Should there be text about only one of service or job can be provided?

Reply:
Done